src/main/resources/translations/program_strings_en.properties

```diff
@@ -341,19 +341,19 @@ xis.dontsave = Don\'t save
 xis.sepcomma = comma (,)
 xis.sepcolon = colon (:)
 xis.sepsemicolon = semicolon (;)
-xis.sephash = hash (#)
+xis.sephash = poundsign (#)
 xis.sepother = other:
 
 ## Copy data tab
-copyd.copyallmetadatasame = Copy the values of all writable tags from the source image to the target image(s), writing the information to same-named tags in the preferred groups
+copyd.copyallmetadatasame = Copy the values of all writable tags from the source image to the target image(s), writing the info to same-named tags in the preferred groups
 copyd.copyallsamegroup = Copy the values of all writable tags from the source image to the target image(s), preserving the original tag groups.
-copyd.copyallmetadatatoxmpformat = Copy all metadata to xmp format
-copyd.copybyselection = Copy metadata using below mentioned selective group options.
-copyd.copyexifcheckbox = Copy exif data (-exif:all)
-copyd.copyxmpcheckbox = Copy xmp data (-xmp:all)
+copyd.copyallmetadatatoxmpformat = Copy all metadata to XMP format
+copyd.copybyselection = Copy metadata using the selective group options mentioned below.
+copyd.copyexifcheckbox = Copy Exif data (-exif:all)
+copyd.copyxmpcheckbox = Copy XMP data (-xmp:all)
 copyd.copyiptccheckbox = Copy IPTC data (-iptc:all)
 copyd.copyicc_profilecheckbox = Copy ICC(_profile) data (-icc_profile:all)
-copyd.copygpscheckbox = Copy gps data (-gps:all)
+copyd.copygpscheckbox = Copy GPS data (-gps:all)
```

Comment:
I have only some idea what
"All metadata writing info to same named tags to preferred groups<br><br>"
and
"All metadata preserving the original tag groups"  means (below).

Reply:
This is directly copied from the ExifTool Program Documentation Pages from Phil. Nowadays you have older, deprecated tag groups or tah groups that have been created by software manaufacturers and which are actually redundant as the same tags exist in the "default" tag groups.. If you want to keep those older deprecated tags you use the "All metadata preserving the original tag groups". 
Otherwise you use the ".. same named tags to preferred groups"